NAMESPACE

```diff
@@ -24,8 +24,10 @@ export(hipercow_environment_exists)
 export(hipercow_environment_list)
 export(hipercow_environment_show)
 export(hipercow_envvars)
+export(hipercow_get_cores)
```

Comment:
hipercow_parallel_cores perhaps?

Reply:
I've gone for the symmetry of
hipercow_parallel_get_cores and 
hipercow_parallel_set_cores